tests/datastore/drivers/redis/RedisStore.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from "chai"
+import { RedisStore } from "../../../../src/datastore"
+import { Container } from "inversify"
+
+describe("RedisStore", () => {
+    let testStore = new RedisStore()
+    let container: Container = new Container()
+    let key = "test-key"
+    let value = "test-value"
+
+    beforeEach(async () => {
+        await testStore.startup(container)
+    })
+
+    afterEach(async () => {
+        await testStore.shutdown(container)
+    })
+
+    describe("startup", () => {
+        it("should register the RedisStore implementation with the container", () => {
+            let store = container.get(RedisStore)
+
+            expect(store).to.equal(testStore)
+            expect(testStore.isHealthy()).to.equal(true)
+        })
+    })
+
+    describe("shutdown", () => {
+        it("should unregister the PG Driver implementation with the container", async () => {
+            await testStore.shutdown(container)
+
+            expect(container.isBound(RedisStore)).to.equal(false)
+            expect(testStore.isHealthy()).to.equal(false)
+        })
+    })
+
+    describe("sendCommand", () => {
+        it("should send commands to redis and get the results", async () => {
```

Comment:
Would it be possible to add a test for a command know to throw an error so we can ensure consistent failure conditions?